src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -99,6 +99,14 @@ public PersonBuilder withMatch(String match) {
         return this;
     }
 
+    /**
+     * Removes any existing match for the {@code Person} that we are building.
+     */
+    public PersonBuilder withoutMatch() {
+        this.match = null;
+        return this;
+    }
```

Comment:
I dont think we need this function, just call withMatch(null) is fine. Maybe just need to update the JavaDoc for withMatch. 

I also do not think you are using it anymore?

Reply:
I think this function is fine, since `withMatch(null)` is not really the intended behaviour of `withMatch()`.